JS/portfolioView.js

```diff
@@ -2,17 +2,15 @@
 var portfolioView = {};
 
 portfolioView.setTeasers = function() {
-  $('.article-body *:nth-of-type').hide();
+  $('.article-body *:nth-of-type(n+2)').hide();
   $('.read-on').on('click', function(event) {
     event.preventDefault();
-    event.stopPropagation();
     if($(this).text() === 'Show Less') {
-      $(this).prev().children('*:nth-of-type').hide();
+      $(this).prev().children('*:nth-of-type(n+2)').hide();
       $(this).text('Read More');
     } else {
       event.preventDefault();
```

Comment:
I'm not sure that you need to repeat the `event.preventDefault();` here, since the first time you call it is outside your `if` loop.